clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1649,6 +1668,33 @@ class SyclKernelDeclCreator : public SyclKernelFieldHandler {
     Params.push_back(NewParam);
   }
 
+  // Obtain an integer value stored in a template parameter of buffer_location
+  // property to pass it to buffer_location kernel attribute
+  void handleXilinxProperty(ParmVarDecl *Param, QualType PropTy,
+                            SourceLocation Loc) {
+    if (!isSyclXilinxType(PropTy))
+      return;
+    /// TODO: when D88645 lands update this code to use that instead.
+    ASTContext &Ctx = SemaRef.getASTContext();
+    const CXXRecordDecl *RD = PropTy->getAsCXXRecordDecl();
+    const CXXRecordDecl *PRD = cast<CXXRecordDecl>(RD->getParent());
```

Comment:
DRY:
```suggestion
    auto &Ctx = SemaRef.getASTContext();
    const auto RD = PropTy->getAsCXXRecordDecl();
    const auto PRD = cast<CXXRecordDecl>(RD->getParent());
```
anyway, does not need to make the change if it is throwaway code...